gamestonk_terminal/jupyter/dashboards/dashboards_controller.py

```diff
@@ -37,49 +35,60 @@ def print_help(self):
         """Print help"""
         help_str = """
    stocks        interactive dashboard with ticker information
+   correlation   interactive dashboard with correlation information
         """
         print(help_str)
 
     def call_stocks(self, other_args: List[str]):
         """Process stocks command"""
-        parser = argparse.ArgumentParser(
-            add_help=False,
-            formatter_class=argparse.ArgumentDefaultsHelpFormatter,
-            prog="stocks",
-            description="""Shows an interactive stock dashboard""",
-        )
-        parser.add_argument(
-            "-j",
-            "--jupyter",
-            action="store_true",
-            default=False,
-            dest="jupyter",
-            help="Shows dashboard in jupyter-lab.",
-        )
+        create_call(other_args, "stocks", "stocks")
 
-        ns_parser = parse_known_args_and_warn(
-            parser, other_args, EXPORT_ONLY_RAW_DATA_ALLOWED
-        )
+    def call_correlation(self, other_args: List[str]):
+        """Process correlation command"""
+        create_call(other_args, "correlation", "correlation")
+
+
+def create_call(other_args: List[str], name: str, filename: str = None) -> None:
+    filename = filename if filename else name
 
-        if ns_parser:
-            cmd = "jupyter-lab" if ns_parser.jupyter else "voila"
-            file = "gamestonk_terminal/jupyter/dashboards/stocks.ipynb"
+    parser = argparse.ArgumentParser(
+        add_help=False,
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+        prog=name,
+        description="""Shows correlations between stocks""",
+    )
+    parser.add_argument(
+        "-j",
+        "--jupyter",
+        action="store_true",
+        default=False,
+        dest="jupyter",
+        help="Shows dashboard in jupyter-lab.",
+    )
 
+    ns_parser = parse_known_args_and_warn(
+        parser, other_args, EXPORT_ONLY_RAW_DATA_ALLOWED
+    )
+
+    if ns_parser:
+        cmd = "jupyter-lab" if ns_parser.jupyter else "voila"
+        file = f"gamestonk_terminal/jupyter/dashboards/{filename}.ipynb"
```

Comment:
This path will probably not work on Windows.
You can use `os.path` or `pathlib` to construct the paths in an OS-agnostic way (search for `os.path.join` in the repo for examples)